<?xml version="1.0" ?>
<launch>
	<!-- Load joint controller configurations from YAML file to parameter server -->
	<rosparam file="$(find yr_description)/config/trajectory_control.yaml" command="load"/>
	<!-- load the controllers -->
	<node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/exo" args="yr_trajectory_controller" />
	<!-- convert joint states to TF transforms for rviz, etc -->
	<!-- <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="false" output="screen"><remap from="/joint_states" to="/uthai/joint_states" /></node> -->
</launch>